sdks/java/io/parquet/src/test/java/org/apache/beam/sdk/io/parquet/ParquetIOTest.java

```diff
@@ -69,6 +76,44 @@
         "Faraday", "Newton", "Bohr", "Galilei", "Maxwell"
       };
 
+  @Test
+  public void testBlockTracker() throws Exception {
+    OffsetRange range = new OffsetRange(0, 1);
+    ParquetIO.ReadFiles.BlockTracker tracker = new ParquetIO.ReadFiles.BlockTracker(range, 7, 3);
+    assertTrue(Math.abs(tracker.getProgress().getWorkRemaining() - 6) < 0.01);
+    assertTrue(Math.abs(tracker.getProgress().getWorkCompleted()) < 0.01);
+    tracker.tryClaim((long) 0);
```

Comment:
you can use `0L` to represent that `0` is a `long`.

Reply:
Noted!